pkg/snapstore/swift_snapstore.go

```diff
@@ -100,14 +100,9 @@ func NewSwiftSnapStore(config *brtypes.SnapstoreConfig) (*SwiftSnapStore, error)
 }
 
 func getClientOpts(isSource bool) (*clientconfig.ClientOpts, error) {
-	if isSource {
-		return &clientconfig.ClientOpts{
-			EnvPrefix: envPrefixSource,
-		}, nil
-	}
-
-	if _, isSet := os.LookupEnv(swiftCredentialJSONFile); isSet {
-		if filename := os.Getenv(swiftCredentialJSONFile); filename != "" {
+	prefix := getEnvPrefixString(isSource)
+	if _, isSet := os.LookupEnv(prefix + swiftCredentialJSONFile); isSet {
+		if filename := os.Getenv(prefix + swiftCredentialJSONFile); filename != "" {
```

Comment:
`os.LookupEnv` already returns the value set against a key in the environment. Is there a reason that you wish to get the value again via `os.Getenv`?

Reply:
yes, you are right ... will fix this at all places in separate  PR.